<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>20</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选择对局方式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Leave">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>890</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>清除棋局</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>980</width>
     <height>980</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 10pt &quot;Microsoft YaHei&quot;;                        
border-radius: 4px;                                            
background-color: argb(60, 60, 60, 130);          
color: rgb(255, 255, 255);                                  </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lab_1">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>10</y>
     <width>230</width>
     <height>300</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lab_2">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>320</y>
     <width>230</width>
     <height>500</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LLLLL">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>60</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>当前下棋玩家：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Big_but1">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>80</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>双人博弈</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Big_but2">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>155</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>四人对决</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Big_but3">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>230</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>六人血拼</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Push_Player">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>60</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>330</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>转动棋盘</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>410</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>智能托管</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
